src/bootstrap/bootstrap.py

```diff
@@ -550,6 +572,10 @@ def rustc(self):
         """Return config path for rustc"""
         return self.program_config('rustc')
 
+    def rustfmt(self):
+        """Return config path for rustfmt"""
+        return self.program_config('rustfmt')
```

Comment:
Should bootstrap.py check the channel and when on beta/stable return None, throw an error, ...?

Reply:
Ah I see what you mean. Download it conditionally based on the channel, and then the rustfmt method would throw an error if invoked. Right?

Reply:
Ah, no, not quite -- we'd always download it if the key is present in stage0.txt, but we should not break if it's not basically :)

Reply:
Ah, OK! Should I still list it as a command and in the help text if we're on a channel where it's unavailable?

Reply:
Ideally, no. But it doesn't matter too much, almost no one is likely going to try so as long as the failure mode is an error message rather than just "rustfmt not found" or something.